requirements.txt

```diff
@@ -2,6 +2,7 @@ azureml-core>=1.15.0
 azureml-dataprep
 azureml-mlflow
 azureml-widgets
+azureml-sdk
```

Comment:
we're intentionally not adding `azureml-sdk` to the `requirements.txt`, please remove. i do not think it is needed in this notebook, if so please simply install the additional azureml-* packages in the notebook

Reply:
Didn't you request moving it from the notebook to requirements.txt in your earlier comment? I'll move it back.

Reply:
I meant the packages in `requirements.txt` **should be** sufficient to run everything in this notebook, without any extra installs. if not, please only install the specific azureml-* packages, i.e. azureml-mlflow, azureml-core, azureml-dataprep, azureml-train 

I suspect no additional installs are needed for this tutorial as long as it's only using azureml.core functionality